.ci/updatecli/bump-logstash-version.yml

```
@@ -0,0 +1,81 @@
+---
+name: Update logstash version files
+pipelineid: "logstash/version-updates"
```

If you run this manifest targeting multiple branches, then the `pipelineid` should be unique. I'd recommend using `LOGSTASH_BRANCH`.

```suggestion
pipelineid: "logstash/version-updates-{{ requiredEnv "LOGSTASH_BRANCH" }}"
```

---

Updated.